WMIAdventure/frontend/docs/pages/creative/CardsCreator/atoms/CardDescribePreview.md

```diff
@@ -20,13 +20,14 @@ class CardDescribePreview extends React.Component {
                         {this.props.cardTooltip}
                     </H3>
                 </Section>
-                <Button onClick={this.props.showDescribeInputsHandler}>
+                <Submit onClick={this.props.showDescribeInputsHandler}>

```

Comment:
Czemu to jest zmieniane?

Reply:
Bo w formie submit powinien być a nie button.